spec/features/target_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'target', :js do
+  let(:user) { FactoryGirl.create :user }
+  let(:topic) { FactoryGirl.create :topic }
+  let!(:target) { FactoryGirl.create :target, user: user }
+
+  it 'sign in' do
```

Comment:
These tests don't belong to the target screen, move it to another file, or rename this one